### PR TITLE
SUBS-1825-release-date

### PR DESCRIPTION
**Pull request by haseeb-gh**
- updated ena core version.
- updated few tests.

### CODE REVIEW

File: src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java [hunk 1]
```diff
@@ -90,6 +91,7 @@ public void whenUploadedFilesHasData_ThenItWillBeInsertedIntoFileMetadata() thro
         study.setId(UUID.randomUUID().toString());
         uk.ac.ebi.subs.data.Submission submission = new uk.ac.ebi.subs.data.Submission();
         submission.setTeam(team);
+        submission.setSubmissionDate(new Date());
```
Comment (ke4) [inline src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java:94]: Is it mandatory to set the `submissionDate` on a submission? What happens if it is null?

**Review by ke4 — COMMENTED** [review]
👍 Just a tiny comment/question to answer.
Thanks, Karel

File: src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java (hunk 1 above)
Comment (haseeb-gh) [reply src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java:94]: at first it was mandatory. thats why i updated the code in this project. but later when i looked the at the ena-validator project i realized this should not be a compulsory. i forgot to revert the code in this project but then i thought its probably not bad thing to keep it around. it kind of makes sense to set a submission date on a submission :)

let me know if you think its not necessary.

Comment (ke4) [reply src/test/java/uk/ac/ebi/subs/ena/EnaAgentAssayDataSubmissionsProcessorTest.java:94]: Ok, it is fine. You can leave it there.

**Review by ke4 — APPROVED** [review]

**Pull request merged**